### PR TITLE
Do not create empty regex.list file

### PR DESCRIPTION
**Pull request by DL6ER**
**By submitting this pull request, I confirm the following:** 
*please fill any appropriate checkboxes, e.g: [X]*

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [X] I have made only one major change in my proposed changes.
- [X] I have commented my proposed changes within the code.
- [X] I have tested my proposed changes, and have included unit tests where possible.
- [X] I am willing to help maintain this change if there are issues with it later.
- [X] I give this submission freely and claim no ownership.
- [X] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [X] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

Please make sure you [Sign Off](https://github.com/pi-hole/pi-hole/wiki/How-to-signoff-your-commits.) all commits. Pi-hole enforces the [DCO](https://github.com/pi-hole/pi-hole/wiki/Contributing-to-the-project).

---

Remove creation of an empty `regex.list` file. We are not using this file any longer.

### CODE REVIEW

File: automated install/basic-install.sh [hunk 1]
```diff
@@ -1923,6 +1917,8 @@ installPihole() {
                 printf "  %b Warning: 'lighty-enable-mod' utility not found\\n" "${INFO}"
                 printf "      Please ensure fastcgi is enabled if you experience issues\\n"
             fi
+            # Restart lighttpd to make the new group membership effective
+            restart_service "lighttpd"
```
Comment (Mcat12) [inline automated install/basic-install.sh]: Lighttpd should already be restarted here, after `installPihole` is run:
https://github.com/pi-hole/pi-hole/blob/bc8976fe755a9f7eebbd63847db68e869389bdb4/automated%20install/basic-install.sh#L2636-L2645

Comment (DL6ER) [reply automated install/basic-install.sh]: Thank you for pointing this out, I'll remove this commit from this PR.

The user reporting the issue mentioned meanwhile that they use `apache2` and have `INSTALL_WEB_SERVER == false` which is a configuration we do not directly support.

**Comment by cottton** [comment]
> Remove creation of an empty `regex.list` file. We are not using this file any longer.

Since when? Next release?
And why? Whats the background?

Im using `Pi-hole Version v4.3.2`
and still got this file and its still in use.

**Comment by PromoFaux** [comment]
in the `development` branch, and the beta branch `release/v5.0`, the various lists stored in `/etc/pihole` are moved into a database. No need to create the file if we are no longer using it.

Once we release 5.0, and you upgrade, your `regex.list` file will be read into the databse file (`/etc/pihole/gravity.db`)

**Pull request merged**